arch/x86/configs/x86_64_defconfig

```diff
@@ -3681,3 +3681,10 @@ CONFIG_FONT_SUPPORT=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
 CONFIG_ARCH_HAS_SG_CHAIN=y
+
+CONFIG_NET_9P=y
+CONFIG_NET_9P_VIRTIO=y
+CONFIG_9P_FS=y
+CONFIG_9P_FS_POSIX_ACL=y
+CONFIG_PCI=y
+CONFIG_VIRTIO_PCI=y
```

Comment:
i like those CONFIG build with kernel modules.

other warning:
arch/x86/configs/x86_64_defconfig:3685:warning: override: reassigning to symbol NET_9P
arch/x86/configs/x86_64_defconfig:3689:warning: override: reassigning to symbol PCI
arch/x86/configs/x86_64_defconfig:3690:warning: override: reassigning to symbol VIRTIO_PCI